### PR TITLE
Data 3764

### PR DESCRIPTION
**Pull request by xenusuk1**
for data-3764

### CODE REVIEW

**Comment by LegacyKing** [comment]
pcgen.persistence.lst.DataLoadTest > testLoadSources[Pathfinder RPG for Players] FAILED
    org.junit.ComparisonFailure: Errors encountered while loading Pathfinder RPG for Players expected:<[]> but was:<[Invalid Token - does not contain a colon: '' in Spell Pocketful of Vipers of Campaign: Ultimate Intrigue; SourceFile: file:/home/travis/build/PCGen/pcgen/data/pathfinder/paizo/roleplaying_game/ultimate_intrigue/ui_spells.lst]>
        at org.junit.Assert.assertEquals(Assert.java:115)
        at pcgen.persistence.lst.DataLoadTest.testLoadSources(DataLoadTest.java:165)
pcgen.persistence.lst.DataLoadTest > testLoadSources[Pathfinder RPG Core Supplements] FAILED
    org.junit.ComparisonFailure: Errors encountered while loading Pathfinder RPG Core Supplements expected:<[]> but was:<[Invalid Token - does not contain a colon: '' in Spell Pocketful of Vipers of Campaign: Ultimate Intrigue; SourceFile: file:/home/travis/build/PCGen/pcgen/data/pathfinder/paizo/roleplaying_game/ultimate_intrigue/ui_spells.lst]>
        at org.junit.Assert.assertEquals(Assert.java:115)
        at pcgen.persistence.lst.DataLoadTest.testLoadSources(DataLoadTest.java:165)

File: data/pathfinder/paizo/roleplaying_game/ultimate_intrigue/ui_spells.lst [hunk 1]
```diff
@@ -7,107 +7,107 @@ SOURCELONG:Ultimate Intrigue	SOURCESHORT:UI	SOURCEWEB:http://paizo.com/products/
...
-Illusion of Treachery (Greater)	CLASSES:Antipaladin,Mesmerist=4|Occultist,Psychic=5|Magus,Sorcerer,Wizard=6										SCHOOL:Illusion			SUBSCHOOL:Figment											TYPE:Arcane.Divine.Psychic		COMPS:S,F		CASTTIME:1 standard action						RANGE:Close							TARGETAREA:You and one creature																DURATION:(CASTERLEVEL) rounds [D]											SAVEINFO:Will disbelief											SPELLRES:Yes			SOURCEPAGE:p.217		DESC:This spell functions as illusion of treachery except that it also conceals your own actions (as illusion of calm) and renders invisible all spell effects and ranged attacks originating from you until they reach the attack's target (this doesn't cause the attack's target to be unable to avoid the attack, as they still see the attack originating from the spell's target).|PRERULE:1,Displa
```
Comment (LegacyKing) [inline data/pathfinder/paizo/roleplaying_game/ultimate_intrigue/ui_spells.lst]: Something was broken here. Check for spaces.

**Comment by xenusuk1** [comment]
updated commit, see commented out old line replaced with new

**Comment by LegacyKing** [comment]
Conflict after the prettyLST run. Can you email the file direct? drew0500@yahoo.com

**Comment by xenusuk1** [comment]
added file as attachment to bug

**Comment by LegacyKing** [comment]
https://github.com/PCGen/pcgen/pull/4408

**Pull request closed without merging**